defi/src/api2/scripts/updateMetadataExtra.js

```diff
@@ -0,0 +1,26 @@
+const metadata = require('../metadata.json')
+const fs = require('fs')
+
+updateItemInfo(metadata.protocols)
+updateItemInfo(metadata.treasuries)
+updateItemInfo(metadata.entities)
+fs.writeFileSync(__dirname + '/../metadata.json', JSON.stringify(metadata))
```

Comment:
better to use METADATA_FILE here?

Reply:
issue here was, importAdapter was taking too long, in ts, it was like 10 minutes on my laptop, with this js, it was 9 seconds

Reply:
I dunno how to get around ts taking so long for each adapter import

Reply:
`METADATA_FILE` constant is in ts, cant be imported :(

Reply:
i see